### PR TITLE
Add maestro test to cover showing about:blank when redirected with empty url

### PR DESCRIPTION
**Pull request by catalinradoiu**
Task/Issue URL: https://app.asana.com/1/137249556945/project/1211724162604201/task/1211992268979253?focus=true

### Description

Added a new security test for address bar spoofing with about:blank empty address and updated the existing download path test to check for "about:blank" instead of "Search". Also added a temporary internal security test workflow that will be removed once the associated task is closed.

### Steps to test this PR

_Security Tests_
- [ ] Run the new security test for address bar spoofing with about:blank empty address (8_-_AddressBarSpoof,_aboutblankemptyaddress)
- [ ] Verify that the download path test correctly checks for "about:blank" in the omnibar (5_-_AddressBarSpoof,_downloadpath)

### CODE REVIEW

**Comment by catalinradoiu** [comment]
* **#7190** <a href="https://app.graphite.com/github/pr/duckduckgo/Android/7190?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.com/github/pr/duckduckgo/Android/7190?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `develop`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: .github/workflows/end-to-end-robintest.yml [hunk 1]
```diff
@@ -129,6 +129,21 @@ jobs:
           workspace: .maestro
           include-tags: securityTest
 
+      ## This workflow will be deleted once we close the https://app.asana.com/1/137249556945/project/1211724162604201/task/1212015085457815?focus=true task
```
Comment (mikescamell) [inline .github/workflows/end-to-end-robintest.yml:132]: ❤️

**Review by mikescamell — APPROVED** [review]
Nice work! :shipit:
